apps/mempool/config.json

```diff
@@ -0,0 +1,64 @@
+{
+   "$schema": "../schema.json",
+   "name": "Mempool",
+   "available": true,
+   "exposable": false,
```

Comment:
Any reason you marked it as non-exposable? I think it should be

Reply:
You are right